app/controllers/users_controller.rb

```diff
@@ -3,7 +3,7 @@ class UsersController < Clearance::UsersController
   private
 
   def user_params
-    params.require(:user).permit(:username, :email, :password)
+    params.require(:user).permit(:avatar, :username, :email, :password)
```

Comment:
so important use the strong params with the permit, for more security, good job 👍🏼